File: c/fun_fact.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    const char *facts[] = {
        "Honig verdirbt nie.",
        "Eine Gruppe von Flamingos wird 'Flamboyance' genannt.",
        "Bananen sind Beeren, aber Erdbeeren sind es nicht.",
        "Der Eiffelturm kann sich im Sommer um bis zu 15 cm dehnen."
    };

    srand(time(0));
    int randomIndex = rand() % 4;
    printf("%s\n", facts[randomIndex]);

    return 0;
}

File: java/fun_fact.java
import java.util.Random;

public class FunFact {
    public static void main(String[] args) {
        String[] facts = {
            "Honig verdirbt nie.",
            "Eine Gruppe von Flamingos wird 'Flamboyance' genannt.",
            "Bananen sind Beeren, aber Erdbeeren sind es nicht.",
            "Der Eiffelturm kann sich im Sommer um bis zu 15 cm dehnen."
        };
        
        Random rand = new Random();
        System.out.println(facts[rand.nextInt(facts.length)]);
    }
}
